packer/scripts/infrasim-compute.sh

```diff
@@ -3,14 +3,17 @@
 export LC_ALL=C
 
 # instal dependency for infrasim-compute
-apt-get -y install socat ipmitool qemu openipmi python-pip libssl-dev libssh-dev libpython-dev libffi-dev sgabios 
+apt-get install -y python-pip libssl-dev libpython-dev git
 
 pip install setuptools
 pip install --upgrade pip
 sleep 1
 
 # install infrasim-compute
-pip install infrasim-compute
+git clone https://github.com/InfraSIM/infrasim-compute.git
```

Comment:
keep source code help user to identify the exact commit of his/her environment
remove source code hide all details to common user

I'm OK with both, just confirm